mods/d2k/rules/arrakis.yaml

```diff
@@ -87,7 +87,7 @@ sandworm:
 		Speed: 42
 		Locomotor: worm
 	Targetable:
-		TargetTypes: Ground, Creep
+		TargetTypes: Ground, Creep, Dangerous
```

Comment:
IMO we should make the naming more explicit. How about `IgnoredByBots`?

Reply:
Renamed.